features/specs/S-en-T-test-omgekeerd-ouders/Tg033.feature

```diff
@@ -378,6 +386,14 @@ Functionaliteit: Tg033 - Paassen-Pauw - Twee gehuwde vrouwen met 4 kinderen
       | minderjarige.burgerservicenummer | 000000036        |
       | ouder.burgerservicenummer        | 000000012        |
       | derde.burgerservicenummer        | 000000024        |
+      | derde.type                       | Derde            |
+    En heeft de persoon een 'gezag' met de volgende gegevens
```

Comment:
zie opmerking hierboven. ik denk dat dit onbekende derde moet zijn want ger. uitspraak 1D. dus heeft m.i. 000000024 geen gezag over 000000048

Reply:
Er mag niet vanuit worden gegaan dat de derde de partner (000000024) is van de ouder (000000012). Hierdoor resulteert de gezag uitspraak in OnbekendeDerde, waarin de partner (000000024) geen rol meer in speelt. Hierdoor vervalt de gezag uitspraak.